### DISCUSSION TITLE
Lepton @ Floorp

### DISCUSSION THREAD

**Discussion by GHNewbiee** [Floorp-Projects/Floorp#1103] [Q&A] [answered]
Hi, I hope that you are aware of [Lepton](https://github.com/black7375/Firefox-UI-Fix) project. You may check if `Lepton` would contribute to `Floorp`. That's all. *Tia*

**Comment by CutterKnife** [comment, marked as answer]
Floorp has a built-in Lepton design settings. Also, the list of Lepton contributors shows that there is a Floorp dev.
